[SQS] Document how native message attributes are promoted to NServiceBus headers (#7066)
Particular/docs.particular.net: transports/sqs/native-integration_native-message-attributes-promotion_sqs_[7.3,).partial.md

### Native message attributes promotion

> When ingesting native messages, e.g., messages sent by a non-NServiceBus sender, any native message SQS attribute will be promoted by the SQS transport to an NServiceBus header.

Reviewer: ```suggestion
When ingesting native messages, e.g., messages sent by a non-NServiceBus sender, the SQS transport will promote all native message SQS attributes to an NServiceBus header.
```

Do we also want to clarify where those will be? Either in the payload or in an SQS native header containing all nsb headers? Or is that out of scope here?

Author: I think it's not in scope here

> [!NOTE]

> > If the promoted native message attribute key matches an existing NServiceBus header, the native message attribute is discarded in favor of the existing NServiceBus header.

Author: I have not aligned docs. Good catch ￼

> > If the promoted native message attribute key matches an existing NServiceBus header, the native message attribute is discarded in favor of the existing NServiceBus header.

Author: ```suggestion
> If the promoted native message attribute key matches an existing NServiceBus header, the native message attribute will override the existing NServiceBus header.
```
